README.md

```diff
@@ -110,15 +110,6 @@ You can also run the code in development with the following command:
 npm run dev
 ```
 
-### Build from Docker
```

Comment:
@Drazair Ups, you have overridden some edits ;-)

Reply:
That was a mistake. But these commands doesn't work on windows, so they might confuse a few people. Feel free to add them again, with either a disclaimer or a windows version